applications/serial_lte_modem/src/twi/slm_at_twi.c

```diff
@@ -28,10 +28,10 @@ static const struct device *slm_twi_dev[] = {
 	DEVICE_DT_GET_OR_NULL(DT_NODELABEL(i2c3)),
 };
 static uint8_t twi_data[TWI_DATA_LEN * 2 + 1];
+static char rsp_buf[256];
```

Comment:
Same comment as in the other file, can't the `data_buf` be used instead?

Reply:
Let's continue to use a local buffer here as it's for AT response and meanwhile the functions make it impossibile to call rsp_send() directly.